dataplane/saiserver/ports.go

```diff
@@ -219,16 +219,32 @@ func (port *port) CreatePort(ctx context.Context, req *saipb.CreatePortRequest)
 			fwdPort := &fwdpb.PortCreateRequest{
 				ContextId: &fwdpb.ContextId{Id: port.dataplane.ID()},
 				Port: &fwdpb.PortDesc{
-					PortType: fwdpb.PortType_PORT_TYPE_GENETLINK,
+					PortType: fwdpb.PortType_PORT_TYPE_TAP,
```

Comment:
Is the above TODO fixed by these changes?

Reply:
no, this todo is going to stay probably until we have a plan for dynamic links